conftest.py

```diff
@@ -98,6 +98,19 @@ def pytest_addoption(parser):
 def token(request):
     return Secret(request.config.option.token)
 
+def get_device_name_in_separate_process():
+    # Needs to be done in the separate process, because
+    # `torch_hpu.get_device_name()` occupies a device and doesn't release it.
+    # For tests what work in a separate process if there's a need for all devices
+    # not all of them will be available due the occupation from current pytest process.
+    import subprocess
+    script = \
+    "import habana_frameworks.torch.hpu as torch_hpu\n"\
+    "print(torch_hpu.get_device_name())"
+
+    result = subprocess.run(f"python -c '{script}'", shell=True, capture_output=True, text=True)
+
+    return result.stdout
```

Comment:
`return results.stdout.strip()` since we check `if not name` by caller (i.e. empty string).

Reply:
That is, on non-gaudi I get:
```python
result = CompletedProcess(args="python -c 'import habana_frameworks.torch.hpu as torch_hpu\nprint(torch_hpu.get_device_name())'", returncode=0, stdout='\n', stderr='/usr/lib/python3.10/inspect.py:288: FutureWarning: `torch.distributed.reduce_op` is deprecated, please use `torch.distributed.ReduceOp` instead\n  return isinstance(object, types.FunctionType)\n/usr/local/lib/python3.10/dist-packages/habana_frameworks/torch/hpu/__init__.py:135: UserWarning: Device not available\n  warnings.warn("Device not available")\n')
```